modules/plugin/geopkg/src/test/java/org/geotools/geopkg/GeoPkgDataStoreFactoryTestSetup.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2022, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.geopkg;
+
+import org.geotools.jdbc.JDBCBooleanTestSetup;
+import org.geotools.jdbc.JDBCTestSetup;
+
+public class GeoPkgDataStoreFactoryTestSetup extends JDBCBooleanTestSetup {
```

Comment:
```suggestion
public class GeoPkgDataStoreFactoryTestSetup extends JDBCDelegatingTestSetup {
```

You'll also have to adjust the methods below.

These two classes show an example of a test that is unique to PostGIS, with no inheritance from classes with their pre-existing setup and tests, you can use it as a pattern:
- https://github.com/geotools/geotools/blob/6ff4135e3ddcd1890ff53aa5a676075afa4af54f/modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostGISHStoreTestSetup.java#L24
- https://github.com/geotools/geotools/blob/6ff4135e3ddcd1890ff53aa5a676075afa4af54f/modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostGISHStoreOnlineTest.java

Reply:
thanks for the pointer, fixed.